src/components/InternalLinkButton/InternalLinkButton.tsx

```diff
@@ -0,0 +1,88 @@
+import { styled } from '@mui/material';
+import Link from 'next/link';
+import React from 'react';
+import ArrowInternalLink from '../../stories/components/svg/ArrowInternalLink';
+
+interface Props {
+  label?: string;
+  icon?: boolean;
+  href: string;
+  className?: string;
+  iconFill?: string;
+  iconWith?: number;
+  iconHeight?: number;
+  iconClassName?: string;
+  background?: string;
+}
+
+const InternalLinkButton: React.FunctionComponent<Props> = ({
+  label,
+  icon,
+  href,
+  className,
+  iconFill = '#5B667E',
+  iconWith = 24,
+  iconHeight = 23,
+  background,
+  iconClassName,
```

Comment:
I recomment to rename the prop `icon` to `showIcon` or something similar to indicate with its name that the prop is a boolean. Also the props `iconFill` should not be there as we can change the `fill` by overriding the styles thanks to the `className` using a selector like `& svg`. the other props: `iconWidth`, `iconHeight` should also be handled using the selector like the fill, the `background` should be applied the same logic otherwise we'll need to do something like `background={isLight ? 'x' : 'y'}` while with CSS selector we can use pure CSS to handle the theme.
Same with `iconClassName`, with this we need to generate the styles outside, get the className and pass to it and if I'm not wrong that is deprecated in MUI

Reply:
you are right, done